src/IncludedCommands/Persistent/pbans.luau

```diff
@@ -51,86 +50,275 @@ local function GetCachedUsername(UserId: number): string
     return UsernameCache[UserId]
 end
 
+local function PagesToTable(Pages: BanHistoryPages)
+    local Items = {}
+    while true do
+        table.insert(Items, Pages:GetCurrentPage())
+        if Pages.IsFinished then
+            break
+        end
+        Pages:AdvanceToNextPageAsync()
+    end
+    return Items
+end
 
+type BanHistoryPage = {
+    DisplayReason: string,
+    PrivateReason: string,
+    StartTime: string,
+    Duration: number,
+    Ban: boolean,
+    PlaceId: number,
+}
```

Comment:
(Formatting note - this should be moved up above functions)

This type doesn't contain the error message, which leaves to typing issues below. Potential alternative type:
```luau
type BanHistoryPage = {
    DisplayReason: string,
    PrivateReason: string,
    StartTime: string,
    Duration: number,
    Ban: boolean,
    PlaceId: number,
} | {
    ErrorMessage: string,
}
```